Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM node:11.13.0-alpine
+
+# create destination directory
+RUN mkdir -p /usr/src/poc-intake
```

Comment:
WORKDIR will create the directory if it doesn't exist. This extra layer is unneeded.